<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-FOPH4DRC" failures="1" tests="1" name="Default test" time="16.336" errors="0" timestamp="2025-03-10T16:24:58 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="15.066">
    <failure type="java.lang.AssertionError" message="expected [benakba fakhreddine] but found [test user benakba 1fakhreddine]">
      <![CDATA[java.lang.AssertionError: expected [benakba fakhreddine] but found [test user benakba 1fakhreddine]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinition.SecondScenario.user_checks_the_functionality_of_login(SecondScenario.java:32)
at âœ½.User checks the functionality of login(file:///C:/Users/svs%20deepak/eclipse-workspace/orangeapplication_pagefactory/src/test/resources/Feature/sample3.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
